Makefile

```diff
@@ -130,15 +130,17 @@ test_unit:
 test_race:
 	@go test -race $(PACKAGES_NOSIMULATION)
 
-test_sim:
-	@echo "Running individual module simulations."
-	@go test $(PACKAGES_SIMTEST) -v
-	@echo "Running full Gaia simulation. This may take several minutes."
-	@echo "Pass the flag 'SimulationSeed' to run with a constant seed."
-	@echo "Pass the flag 'SimulationNumKeys' to run with the specified number of keys."
-	@echo "Pass the flag 'SimulationNumBlocks' to run with the specified number of blocks."
-	@echo "Pass the flag 'SimulationBlockSize' to run with the specified block size (operations per block)."
-	@go test ./cmd/gaia/app -run TestFullGaiaSimulation -SimulationEnabled=true -SimulationBlockSize=200 -v
+test_sim_modules:
+	@echo "Running individual module simulations..."
+	@go test $(PACKAGES_SIMTEST)
+
+test_sim_gaia_fast:
```

Comment:
I'm not able to send a successful `SIGINT` to this command.

Reply:
Good point! We should resolve this, but I strongly think we should get this PR merged, and fix this once its merged. Reviewing / rebasing / etc. is becoming a pain, and it will be much easier to parallelize work once its on develop.

Reply:
Just send `SIGKILL`!